app/src/ui/history/expandable-commit-summary.tsx

```diff
@@ -387,20 +388,65 @@ export class ExpandableCommitSummary extends React.Component<
     )
   }
 
-  private renderAuthors = () => {
-    const { selectedCommits, repository } = this.props
+  private renderExpandedAuthor(user: IAvatarUser): string | JSX.Element {
+    if (!user) {
+      return 'Unknown user'
+    }
+
+    if (user.name) {
+      return (
+        <>
+          {user.name}
+          {' <'}
+          {user.email}
+          {'>'}
+        </>
+      )
+    }
+
+    return user.email
+  }
+
+  private renderAuthorList = () => {
     const { avatarUsers } = this.state
-    if (selectedCommits.length > 1) {
-      return
+    const elems = []
+
+    for (let i = 0; i < avatarUsers.length; i++) {
+      elems.push(
+        <div className="author selectable" key={i}>
+          <Avatar user={avatarUsers[i]} title={null} />
+          <div>{this.renderExpandedAuthor(avatarUsers[i])}</div>
+        </div>
+      )
     }
 
+    return elems
+  }
```

Comment:
Nit: I think this could be just a `map`
```suggestion
  private renderAuthorList = () => {
    return this.state.avatarUsers.map((avatar, i) => (
      <div className="author selectable" key={i}>
        <Avatar user={avatar} title={null} />
        <div>{this.renderExpandedAuthor(avatar)}</div>
      </div>
    ))
  }
```

Reply:
Updated to map. Thanks for the call out. I think I just copied this code from the avatar component.